docs/updating.md

```diff
@@ -1057,12 +1057,13 @@ pages for more details on the input and output variable types.
 #### plantcv.threshold.saturation
 
 * pre v3.8: NA
-* post v3.8: bin_img = **plantcv.threshold.saturation**(*rgb_img, threshold=255, channel="any")
+* post v3.8: bin_img = **plantcv.threshold.saturation**(*rgb_img, threshold=255, channel="any"*)
 
-#### plantcv.threshold.texture_filter
+#### plantcv.threshold.texture
```

Comment:
the `texture` function name was mismatched here. I took a look at the git history and it looks like it's always been mismatched, so i changed it here. happy to adjust if needed